config/initializers/social_share_button.rb

```diff
@@ -0,0 +1,3 @@
+SocialShareButton.configure do |config|
+  config.allow_sites = %w(twitter facebook google_plus google_bookmark delicious tumblr plurk pinterest email linkedin)
```

Comment:
What do you think about paring this down to Facebook, Twitter, Pinterest and LinkedIn?